src/Compatibility/Core/src/Compatibility.csproj

```diff
@@ -9,7 +9,7 @@
     <iOSRoot>iOS\</iOSRoot>
     <WindowsRoot>Windows\</WindowsRoot>
     <TizenRoot>Tizen\</TizenRoot>
-    <IsAotCompatible Condition="!$(TargetFramework.StartsWith('netstandard'))">true</IsAotCompatible>
+    <IsAotCompatible Condition="!$(TargetFramework.StartsWith('netstandard')) and $([MSBuild]::GetTargetPlatformIdentifier('$(TargetFramework)')) != 'windows'">true</IsAotCompatible>
```

Comment:
Why is this change needed?

Reply:
Yeah, I didn't want to fix 150+ errors about missing partials then revert it all when RTM was released, so I'm disabling it for now.

Reply:
> "then revert it all when RTM was released"

I feel like there might be a misunderstanding here, that requirement is absolutely **not** going away when .NET 9 goes GA. I do have plans to remove that, but that's a very long term thing that needs some C# features that don't even exist yet (extension interfaces, specifically), so that won't happen before .NET 11/C# 15 at the earliest. Until then, the requirement of marking types marshalled to WinRT as partial will remain there 😅

Or did you mean something else by that? Just making sure the long term plan for CsWinRT is clear.

Reply:
Ah, OK. I misunderstood that the current analyzers were overly verbose and some combination of .NET 9 + new packages would fix that. 

It's uh... been a bit difficult for us to keep track of what's being fixed/changed with certain packages/SDKs 😋